pkg/apis/migration/v1alpha1/directvolumemigration_types.go

```diff
@@ -117,3 +117,9 @@ func (r *DirectVolumeMigration) HasErrors() bool {
 func init() {
 	SchemeBuilder.Register(&DirectVolumeMigration{}, &DirectVolumeMigrationList{})
 }
+
+func (r *DirectVolumeMigration) DirectVolumeMigrationLabels() map[string]string {
+	labels := r.GetCorrelationLabels()
+	labels["directvolumemigration"] = string(r.UID)
```

Comment:
Are lines 122 and 123 doing the same thing?


Correlation label looks like this:

```
// Build label (key, value) used to correlate CRs.
// Format: <kind>: <uid>.  The <uid> should be the ObjectMeta.UID
func CorrelationLabel(r interface{}, uid types.UID) (key, value string) {
	return labelKey(r), string(uid)
}
```

`"directvolumemigration": "<uid>"`

Reply:
Thank you @djwhatle , I have made the appropriate changes.